xbdd-web/src/modules/summary/SummaryContainer.jsx

```diff
@@ -76,7 +76,7 @@ class SummaryContainer extends Component {
             <Grid container>
               <Grid item xs={12}>
                 <Typography variant="h3" className={this.props.classes.summaryTitle}>
-                  G'day Mate
+                  Welcome to XBDD
```

Comment:
I would prefer a relatable gif if you have one.

Reply:
You will have to wait for another PR for that.

Reply:
Fine